dataland-frontend/src/services/ApiClients.ts

```diff
@@ -76,7 +146,7 @@ export class ApiClientProvider {
   }
 
   async getDocumentControllerApi(): Promise<DocumentControllerApi> {
-    return this.getConstructedDocumentManager(DocumentControllerApi);
+    return this.getConstructedDocumentManager(DocumentControllerApi); // TODO why only one without the basePath and ApiKeyController?
```

Comment:
@MarcTM01 Could you please have a look at this TODO review question from Emanuel. I have no knowledge on the matter.

Reply:
The `basePath` here is not needed because someone created a custom function `getConstructedDocumentManager`. However, I do not understand why we are not just using the getConstructedApi function with a corresponding basePath....

Reply:
ok